modules/packages/Socket.chpl

```diff
@@ -698,7 +698,7 @@ proc tcpListener.accept(in timeout: struct_timeval = indefiniteTimeout):tcpConn
       if timeout.tv_sec:c_long != -1 {
         var totalTimeout = timeout.tv_sec:c_long*1000000 + timeout.tv_usec:c_long;
         // timer didn't elapsed
-        if totalTimeout > t.elapsed(TimeUnits.microseconds) {
+        if totalTimeout > (t.elapsed()*1_000_000) {
```

Comment:
It would be cool if we could have a constant for this e.g., `microsecondsPerSec = 1_000_000`. Actually nevermind, I see we just use 1000000 everywhere else already...